Simplified/NYPLBookRegistry.m

```diff
@@ -392,7 +392,7 @@ - (void)syncWithStandardAlertsOnCompletion
 
 - (void)addBook:(NYPLBook *const)book
        location:(NYPLBookLocation *const)location
-          state:(NYPLBookState)state
+          state:(NSInteger)state
```

Comment:
hmm not sure why this changed, but we lost some expressiveness here. If you need an Integer, can't you cast it inside the method instead?

Reply:
Having NYPLBookState as one of it's parameter caused this method not being compiled to the bridging header (not able to be called in Swift).
Now there is only called in one Swift file which is a test, but there might be more in the future.